### PR TITLE
Worked on the notification for the optimizer pool

### PR DESCRIPTION
**Pull request by Sanjeev1308**
Stored the data in local storage and manage notification from there
Will be dismiss after dismiss button clicks
Will be regenrate every two weeks from the dismiss date

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201822812019113

### CODE REVIEW

File: Layout/Shared/Notification/constants.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const OPTIMIZER = "optimizer";
+export const NOTIFICATION_TYPE = "localStorage";
+
+export const messageForDissmissNotification = {
+  optimizer: {
+    name: "Optimizer Pools",
+    description:
+      "Some exciting is on optimizer pools, please check on the compound and earn page!",
```
Comment (ghost) [inline Layout/Shared/Notification/constants.js]: This description sounds odd, did bigwampa provided the text?

Comment (Sanjeev1308) [reply Layout/Shared/Notification/constants.js]: Need to work on content, going to check the adjacent ticket assign to Ncookie for content 

> This description sounds odd, did bigwampa provided the text?

**Review by ghost — CHANGES_REQUESTED** [review]
image
I'm getting this error when I click at the notifications icon (wallet connected).

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**